cirq/testing/logs.py

```diff
@@ -20,7 +20,8 @@
 def assert_logs(
     *matches: str,
     count: Optional[int] = 1,
-    level: int = logging.WARNING,
+    min_level: int = logging.WARNING,
```

Comment:
Does `level` need to go through deprecation?

Reply:
I was on a fence about this. I guess especially for vendor modules treating `cirq.testing` as part of the library is nice.

Reply:
Done.